Vegeting/Global/Manager/FirebaseStorageManager.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  FirebaseStorageManager.swift
+//  Vegeting
+//
+//  Created by kelly on 2022/11/15.
+//
+
+import Foundation
+import FirebaseStorage
+import UIKit
+
+class FirebaseStorageManager {
+    static func uploadImage(image: UIImage, folderName: String, completion: @escaping (URL?) -> Void) {
+        guard let scaledImage = image.scaledToSafeUploadSize,
```

Comment:
p3;
escaping handler는 결국, 비동기 처리를 위해서 사용되는건데 에러를 리턴할때 다소 어려운 점이 있었어요.
그를 보완하기 위해 Result문법을 도입했습니다!
https://developer.apple.com/documentation/swift/result
해당 타입은, 성공과 에러를 처리하기에 가독성 좋고, 이해하기 쉬운 형태로 제공됩니다.
FirebaseManager에 제가 사용한 escaping handler 방식을 보시면 더욱 좋을것 같습니다.
```suggestion
    static func uploadImage(image: UIImage, folderName: String, completion: @escaping (Result<URL, Error>) -> Void) {
        guard let scaledImage = image.scaledToSafeUploadSize,
```

Reply:
[f7cab87](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/141/commits/f7cab8772309edb8520e28fd9ac3c7428c0659ac)
적용해보았습니다! 보시고 피드백 부탁드려요!, download 함수는 다운로드 함수 부분에서 수정하겠습니다!